packages/service-clients/end-to-end-tests/azure-client/src/test/containerCreate.spec.ts

```diff
@@ -710,4 +710,24 @@ describe("Container create in tree-only mode", () => {
 
 		assert(SharedTree.is(container.initialObjects.tree));
 	});
+
+	it("throws if invalid schema is encountered", async function () {
+		const client = createClient();
+		const schema = {
+			initialObjects: {
+				tree: SharedTree,
+				map: SharedMap,
+			},
+		};
+
+		await assert.rejects(
+			async () => client.createContainer(schema, "2"),
```

Comment:
Don't necessarily need to wrap this when using assert.rejects, but fine either way
```suggestion
			client.createContainer(schema, "2"),
```